renovate-presets/default.json5

```diff
@@ -1,7 +1,7 @@
 {
   $schema: 'https://docs.renovatebot.com/renovate-schema.json',
   extends: ['mergeConfidence:all-badges', 'group:monorepos'],
-  dependencyDashboard: true,
+  dependencyDashboard: false,
```

Comment:
What is the reason for this? This is quite useful to open PRs before their schedule time and to open patch updates for the non-main branch for example.

Reply:
I remove this commit from the change.